examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2466,6 +2471,122 @@ impl FrontendNode {
         expect_matches!(result, Err(_));
         Ok(())
     }
+
+    fn test_pseudo_node_privilege(&mut self) -> TestResult {
+        // gRPC server pseudo node can be created even with Top confidentiality.
```

Comment:
Might be nice to split these into separate tests, so if something goes wrong it's easier to investigate.  

(Also, there are test regexes that get skipped when `abitest/module_0/tests/integration_test.rs` gets run, which might be worth aligning the test names with.)

Reply:
Done.